### PR TITLE
Update for EDHOC version 20

### PR DESCRIPTION
**Pull request by actyp**
Incorporate changes needed for supporting EDHOC version 20 (as defined in the changes of `cf-edhoc` library), also bump the commit hash of `cf-edhoc` library and correct the `cf-edhoc.patch`.

### CODE REVIEW

File: scripts/cf-edhoc.patch [hunk 1]
```diff
@@ -167,21 +167,21 @@ index a255c4bd1..de3353efe 100644
 +	public static byte[] generateSharedSecret(OneKey privateKey, OneKey publicKey) {
  
  		if (privateKey == null || publicKey == null) {
- 			System.err.println("Public key and/or privat key not found.");
+ 			System.err.println("Public key and/or private key not found.");
```
Comment (kostis) [inline scripts/cf-edhoc.patch:170]: Perhaps we should communicate (at least) this typo fix to the cf-edhoc repo.

**Review by kostis — APPROVED** [review]
It is OK.  Go ahead and merge.

File: scripts/cf-edhoc.patch (hunk 1 above)
Comment (actyp) [reply scripts/cf-edhoc.patch:170]: They have updated it, the removal/addition is not manual. The previous commit had the typo, while the current does not have it. This diff is due to the the commit update.

Comment (kostis) [reply scripts/cf-edhoc.patch:170]: Ah, OK.

**Pull request merged**